io.github.magnush.mhWaveEdit.yml

```diff
@@ -0,0 +1,107 @@
+# io.github.magnush.mhWaveEdit.yml
+app-id: io.github.magnush.mhWaveEdit
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: mhwaveedit
+
+#separate-locales: false
+
+rename-desktop-file: mhwaveedit.desktop
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-music
+  - --socket=pulseaudio
+  
+modules:
+
+
+- name: gtk2
+  cleanup:
+  - "/bin"
+  - "/share/gtk-2.0"
+  - "/share/aclocal"
+  - "/share/gtk-doc"
+  - "/lib/pkgconfig"
+  - "/lib/gtk-2.0/include"
+  - "/include"
+  - "*.la"
+  x-cpe:
+    product: gtk+
+  config-opts:
+  - "--disable-dependency-tracking"
+  - "--disable-gtk-doc-html"
+  - "--disable-introspection"
+  - "--with-xinput=xfree"
+  sources:
+  - type: archive
+    url: https://download.gnome.org/sources/gtk+/2.24/gtk+-2.24.33.tar.xz
+    sha256: ac2ac757f5942d318a311a54b0c80b5ef295f299c2a73c632f6bfb1ff49cc6da
+    
+    
+
+# gnome themes extra needs intltool>0.4 on Sdk 20.08 
+- name: intltool
+  cleanup:
+  - "*"
+  sources:
+  - type: archive
+    url: https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz
+    sha256: 67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd
+
+- name: gnome-themes-extra
```

Comment:
if you use gtk2 from shared-modules you don't need this.

Reply:
removed.